pkgs.nix

```diff
@@ -1,5 +1,6 @@
 import (
   # let rev = "60cce7e5e1fdf62421ef6d4184ee399b46209366"; in
-  let rev = "04a48a2fcd2370f4e8a8c9fd3394531ccdb30f1e"; in
+  # let rev = "04a48a2fcd2370f4e8a8c9fd3394531ccdb30f1e"; in
+  let rev = "53caacaf56640d04180775aee016d2f16d6f083c"; in
```

Comment:
Which revision is this again? Are we updating this to the latest?

Reply:
I picked the latest pkgs commit when I started on this issue.

Reply:
You should usually get the commit hash matching the exact merge commit. Which would be this one: https://github.com/NixOS/nixpkgs/commit/3d64b6067f1c457f889e91c59ced273be1d1e4f1

Notice you can see it here:

![image](https://user-images.githubusercontent.com/640797/128017469-1c8828eb-788b-473e-8d3e-d5079707f2af.png)

Reply:
That's for in the future, which avoids bringing in other confounding variables/changes.

Reply:
Ah, I'll keep that in mind.